content/reference-rippled.md

```diff
@@ -737,6 +741,181 @@ The response follows the [standard format](#response-formatting), with a success
 
 
 
+## account_channels
+[[Source]<br>](https://github.com/ripple/rippled/blob/release/src/ripple/rpc/handlers/AccountChannels.cpp "Source")
+
+_(Requires the [PayChan amendment](concept-amendments.html#paychan) to be enabled. [New in: rippled 0.33.0][])_
+
+The `account_channels` method returns information about an account's Payment Channels. This includes only channels where the specified account is the source, not the destination. All information retrieved is relative to a particular version of the ledger.
+
+#### Request Format
+An example of the request format:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```json
+{
+  "id": 1,
+  "command": "account_channels",
+  "account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
+  "destination_account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+  "ledger_index": "validated"
+}
+```
+
+*JSON-RPC*
+
+```json
+{
+    "method": "account_channels",
+    "params": [{
+        "account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
+        "destination_account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+        "ledger_index": "validated"
+    }]
+}
+```
+
+*Commandline*
+
+```bash
+#Syntax: account_channels <account> [<destination_account>] [<ledger>]
+rippled account_channels rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn validated
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+The request includes the following parameters:
+
+| Field                 | Type                                       | Description |
+|:----------------------|:-------------------------------------------|:--------|
+| `account`             | String                                     | A unique identifier for the account whose channels to retrieve. Typically this is the account's [Address][]. |
+| `destination_account` | String                                     | _(Optional)_ If provided, filter results to payment channels where the destination of the channel. Typically this is an [Address][]. |
+| `ledger_hash`         | String                                     | _(Optional)_ A 20-byte hex string for the ledger version to use. (See [Specifying a Ledger](#specifying-ledgers)) |
+| `ledger_index`        | String or Unsigned Integer                 | _(Optional)_ The sequence number of the ledger to use, or a shortcut string to choose a ledger automatically. (See [Specifying a Ledger](#specifying-ledgers)) |
+| `limit`               | Integer                                    | _(Optional)_ Limit the number of transactions to retrieve. The server is not required to honor this value. The default is 200. Cannot be smaller than 10 or larger than 400. |
```

Comment:
I rewrote this and other `limit` parameter descriptions to be a little more consistent, but I kept them as separate sentences so that we can be consistent with "Defaults to" instead of "The default is"